.github/workflows/ci.yml

```diff
@@ -71,7 +71,7 @@ jobs:
         ./run_tests.sh
     - name: Coveralls Parallel
       run: |
-        coveralls
+        coveralls || true
```

Comment:
Maybe allow it to fail just if  "GITHUB_TOKEN is not set" ??

Reply:
Hopefully #1794 takes care of this.

`GITHUB_TOKEN` is apparently the only secret available to pull requests in a forked repo.

https://docs.github.com/en/actions/configuring-and-managing-workflows/creating-and-storing-encrypted-secrets#using-encrypted-secrets-in-a-workflow
https://docs.github.com/en/actions/configuring-and-managing-workflows/authenticating-with-the-github_token#permissions-for-the-github_token